test/infrastructure/docker/e2e/docker_test.go

```diff
@@ -129,22 +129,13 @@ var _ = Describe("Docker", func() {
 				}
 				framework.WaitForClusterToProvision(ctx, assertClusterProvisionsInput)
 
-				// Create the workload nodes
-				createMachineDeploymentinput := framework.CreateMachineDeploymentInput{
-					Creator:                 client,
-					MachineDeployment:       md,
-					BootstrapConfigTemplate: bootstrapTemplate,
-					InfraMachineTemplate:    infraTemplate,
-				}
-				framework.CreateMachineDeployment(ctx, createMachineDeploymentinput)
-
-				// Wait for the controlplane nodes to exist
-				assertKubeadmControlPlaneNodesExistInput := framework.WaitForKubeadmControlPlaneMachinesToExistInput{
+				// Wait for at least one control plane node to be ready
+				waitForOneKubeadmControlPlaneMachineToExistInput := framework.WaitForOneKubeadmControlPlaneMachineToExistInput{
 					Lister:       client,
 					Cluster:      cluster,
 					ControlPlane: controlPlane,
 				}
-				framework.WaitForKubeadmControlPlaneMachinesToExist(ctx, assertKubeadmControlPlaneNodesExistInput, "10m", "10s")
+				framework.WaitForOneKubeadmControlPlaneMachineToExist(ctx, waitForOneKubeadmControlPlaneMachineToExistInput)
```

Comment:
I'm not too familiar with the e2e framework – is it worth asserting anything about the KCP's status here?

Reply:
ah ha. That comes later on and I totally forgot to replace the //TODO! Thanks!